### ISSUE TITLE
Docs + Examples

### ISSUE THREAD

**Issue by uprokevin** [ray-project/deltacat#196]
Just wondering if you are hosting some docs somewhere
as well as a a couple of use case examples ?

Thanks

**Comment by raghumdani** [comment]
@pdames Do have example docs? Meanwhile you can refer examples located in our [tests](https://github.com/ray-project/deltacat/tree/main/deltacat/tests/local_deltacat_storage).

**Comment by pdames** [comment]
Hi @pprokevin - apologies for the late response here, but I'm actively working on an initial implementation of the DeltaCAT APIs for Apache Iceberg which should also come with (1) doc updates and (2) examples that will work either locally on your laptop or with AWS Glue. I'll update this issue as these features are released!

**Comment by jhchee** [comment]
@pdames Any update?

**Comment by legout** [comment]
Any news here? Some docs and examples would be very useful.

**Comment by pdames** [comment]
Hi @legout - we're actively working on getting docs and examples stood up in DeltaCAT V2 at https://github.com/ray-project/deltacat/tree/2.0. Once this branch is merged into main, it should contain fully functional docs and examples!
